RecoTracker/DeDx/python/dedxEstimators_cff.py

```diff
@@ -26,24 +26,13 @@
     lowPtTracksDeDxThreshold = cms.double(3.5), # threshold on tracks
 )
 
-dedxHarmonic2 = cms.EDProducer("DeDxEstimatorProducer",
-    tracks                     = cms.InputTag("generalTracks"),
- 
-    estimator      = cms.string('generic'),
-    fraction       = cms.double(0.4),        #Used only if estimator='truncated'
-    exponent       = cms.double(-2.0),       #Used only if estimator='generic'
- 
-    UseStrip       = cms.bool(True),
-    UsePixel       = cms.bool(False),
-    ShapeTest      = cms.bool(True),
-    MeVperADCStrip = cms.double(3.61e-06*265),
-    MeVperADCPixel = cms.double(3.61e-06),
-
-    Reccord            = cms.string("SiStripDeDxMip_3D_Rcd"), #used only for discriminators : estimators='productDiscrim' or 'btagDiscrim' or 'smirnovDiscrim' or 'asmirnovDiscrim'
-    ProbabilityMode    = cms.string("Accumulation"),          #used only for discriminators : estimators='productDiscrim' or 'btagDiscrim' or 'smirnovDiscrim' or 'asmirnovDiscrim'
-
-    UseCalibration  = cms.bool(False),
-    calibrationPath = cms.string(""),
+import RecoTracker.DeDx.DeDxEstimatorProducer_cfi as _mod
+
+dedxHarmonic2 = _mod.DeDxEstimatorProducer.clone(
```

Comment:
Since all parameters below depend on the `estimator` used, I would let here explicitly written in the config also which actually is that estimator, even if it is unchanged wrt to the one implicitly defined in the fillDescriptions: 
```PYTHON
    estimator      = "generic",
```

Reply:
@jeongeun please let us know if you plan to update this

Reply:
Sorry for the late update on this. I will update it.